.github/workflows/deploy-experimental.yml

```diff
@@ -174,6 +174,8 @@ jobs:
       DB_USER: eregs
       DB_PASSWORD: sgere
       DB_PORT: 5432
+      HTTP_AUTH_USER: eregs
+      HTTP_AUTH_PASSWORD: password
     services:
```

Comment:
what is this used for?

Reply:
Dummy environment variables for the unit test to check authentication on the endpoint.  If they do not have proper credentials they cannot access the endpoint.